### DISCUSSION TITLE
How to modify ListBoxItem font properties?

### DISCUSSION THREAD

**Discussion by IAbstract** [AvaloniaUI/Avalonia#6478] [Q&A]
The markup is as follows:

```
          <ListBox Grid.Row="1"
                   FontFamily="Courier"
                   Background="White"
                   Foreground="#424249"
                   VerticalAlignment="Stretch">
            <ListBox.Styles>
              <Style Selector="ListBoxItem">
                <Setter Property="FontSize" Value="12"/>
              </Style>
            </ListBox.Styles>

            <ListBoxItem Content="Default" />
          </ListBox>
```

It doesn't matter what value I set the `FontSize` property, the font size does not change. I tried applying the same `FontSize` attribute to the `ListBox `and directly to the `ListBoxItem`.  

**Update**  
I managed to figure out how to change the `FontSize`:
```
            <ListBox.ItemTemplate>
              <DataTemplate>
                <Label Content="{Binding}"
                       FontSize="12"
                       Height="22"/>
              </DataTemplate>
            </ListBox.ItemTemplate>
```  

... and that worked. However, the individual item container hosting the `Label `is still 10x too tall. I'd like to be able to get more than 3 or 4 items listed in 2" of display area. This is done in WPF using similar methods to what I did here. If `ListBox `is not the right control, let me know. I'm trying to achieve a kind of Explorer look with a field name, created date, item count, etc. So I'm also trying to figure out how to make columns.

![test_portfolios_list](https://user-images.githubusercontent.com/987431/131190222-abaf1fe1-483a-435b-b269-1b83aa22fc6d.PNG)

**Comment by maxkatz6** [comment]
You most likely need to change Padding of ListBoxItem too.
